### ISSUE TITLE
CLI to remove EPUB from Thorium

### ISSUE THREAD

**Issue by PhilippeBruno** [edrlab/thorium-reader#1131]
If there is a CLI to import an EPUB into Thorium, there should be a CLI to remove an EPUB from Thorium. Wildcards should be accepted as well.

Just this afternoon, I had 53 EPUBs to delete from Thorium and I wished I could have called the executable with --delete * to remove all of them at once.

**Comment by danielweck** [comment]
`rm -rf /PATH/TO/EDRLab.Thorium/`  ;)

Joking aside, the current CLI includes a `read` command which takes a publication "title", so in principle a "delete" command is feasible. That being said, I personally think that the "title" fuzzy matching is error prone, I am not sure I would want to rely on it for irreversibly deleting publication.

https://github.com/edrlab/thorium-reader/blob/develop/README.md#command-line

**Comment by PhilippeBruno** [comment]
> https://github.com/edrlab/thorium-reader/blob/develop/README.md#command-line

@danielweck I already checked that link, but I did not find any mention of the -rf command.

**Comment by panaC** [comment]
I agree with @danielweck , we cannot remove a publication with only the title, i think. 
we would have to list the publications with their id to delete them.

**Comment by PhilippeBruno** [comment]
@panaC - I am afraid I beg to differ. Firstly, nowadays, CLI are not aimed at the general user. They are aimed at the advanced user who need to perform advanced tasks, automate time consuming tasks, or perhaps call the application from another application.

Secondly, one could very well damage or even wipe off Thorium's installation by issuing powerful OS commands (that is, operating system commands, not Thorium commands) at the command line using wildcards and you could not even provide additional safeguards as a Thorium developer.

**Comment by danielweck** [comment]
Related issue: https://github.com/edrlab/thorium-reader/issues/1367

**Comment by danielweck** [comment]
I am moving this to a "discussion" because Thorium's CLI is due an audit / technical review soon. There are related issues with URL handling (custom protocol handler), discrepancies between Mac, Linux and Windows, etc.
